sktime/forecasting/base/_base.py

```diff
@@ -360,6 +360,13 @@ def predict_quantiles(self, fh=None, X=None, alpha=None):
             Row index is fh. Entries are quantile forecasts, for var in col index,
                 at quantile probability in second col index, for the row index.
         """
+        if not self.get_tag("capability:pred_int"):
```

Comment:
is this tag now valid for both `predict_interval` and `predict_quantiles`? So if one of both is implemented as a private method in a child, then this tag is `True`? is this documented anywhere what this tag is doing now?

Reply:
maybe lets rename the tag so that its clear that its covering both cases? E.g. `pred_int_or_quantiles`

Reply:
> So if one of both is implemented as a private method in a child, then this tag is True? is this documented anywhere what this tag is doing now?

Agreed to adding that info in the tag. Yes, it now stands for "is one of the interval-like methods implemented".
Regarding renaming, I was going to avoid it this close to a new version since that would be a minor interface change but still one (e.g., people referring to the tag)

Reply:
fixed in https://github.com/alan-turing-institute/sktime/pull/1961